app/routes/login/login.controller.js

```diff
@@ -18,7 +20,7 @@ function LoginController(urlParams){
 	// To bind elements
 	function link(){
 		form = $('form');
-
+		customErrorMsg = $('#loginError');
```

Comment:
This is not a message. It is an element. So it should be customErrorElement or errorEl or etc.